packages/devtools_app/lib/src/shared/feature_flags.dart

```diff
@@ -12,14 +16,22 @@
 ///   "args": [
 ///     "--dart-define=enable_experiments=true"
 ///   ]
-const bool _kEnableExperiments = bool.fromEnvironment('enable_experiments');
+
+bool _kEnableExperiments =
+    const bool.fromEnvironment('enable_experiments') || !kReleaseMode;
+
+/// If true, features, ready for beta testing, will be on.
+///
+/// Always true when [_kEnableExperiments] is true.
+/// See [_kEnableExperiments] documentation  for usage.
+late final bool _kEnableBeta = !isExternalBuild;
```

Comment:
I think this be `const bool.fromEnvironment('enable_experiments') || !isExternalBuild` according to the dart doc

Reply:
Fixed. Thank you. It cannot be constant though.